core/src/com/unciv/ui/tilegroups/TileGroup.kt

```diff
@@ -579,14 +579,20 @@ open class TileGroup(
 
     private fun updateTileColor(isViewable: Boolean) {
         val baseTerrainColor = when {
+            isViewable && tileInfo.isPillaged() && tileSetStrings.tileSetConfig.useColorAsBaseTerrain -> tileInfo.getBaseTerrain().getColor().lerp(Color.BROWN, 0.6f)
+            isViewable && tileInfo.isPillaged() -> Color.WHITE.cpy().lerp(Color.BROWN, 0.6f)
```

Comment:
I'm not 100% sure I like this, imo pillaged sprites should be implemented via the tileset and not directly in the code. For now it's fine, but it's something that'd be nice to change in a future PR.

Reply:
Right. In the future it'd be nice if this was handled purely in the tileset. Although I do like coloring the base tile since Roads are impossible to show.